nulib/next-gen-front-end-react
# 2046 media work types

Now the React Media Player package (https://github.com/nulib/react-media-player | https://www.npmjs.com/package/@nulib/react-media-player) is getting pulled into DC.  And the Work page doesn't error out handling Audio/Video.

### To test
Do a fresh `npm install` to pull in the player package.

Fire up DC against Staging
```
npm run start:use-staging-data
```

Search for an Audio/Video work which exists on Staging.  Click into the search result and the page should render without errors.

![image](https://user-images.githubusercontent.com/3020266/136409387-b9fd8f22-f764-42ee-bc9a-c116f7d6e140.png)
